packages/deployer/test/deploySolutionItems.test.ts

```diff
@@ -1216,359 +1216,359 @@ describe("Module `deploySolutionItems`", () => {
-          .then(() => {
-            done();
-          }, done.fail);
-      });
-
-      it("should handle error on update", done => {
-        const _templates: any[] = [
-          {
-            type: "Group",
-            itemId: "NEWABC123",
-            dependencies: []
-          },
-          {
-            type: "Workforce Project",
-            itemId: "NEW123ABC",
-            dependencies: []
-          }
-        ];
-
-        const templateDictionary: any = {
-          unresolved: {
-            itemId: "resolved"
-          }
-        };
-
-        const workforceData: any = {
-          unresolvedVariable: "{{unresolved.itemId}}"
-        };
-
-        const clonedSolutionsResponse: common.ICreateItemFromTemplateResponse[] = [
-          {
-            id: "NEWABC123",
-            type: "Group",
-            postProcess: false
-          },
-          {
-            id: "NEW123ABC",
-            type: "Workforce Project",
-            postProcess: true
-          }
-        ];
-
-        fetchMock
-          .post(
-            utils.PORTAL_SUBSET.restUrl + "/content/items/NEW123ABC/data",
-            workforceData
-          )
-          .post(
-            utils.PORTAL_SUBSET.restUrl +
-              "/content/users/casey/items/NEW123ABC/update",
-            mockItems.get400Failure()
-          );
-
-        deploySolution
-          .postProcessDependencies(
-            _templates,
-            clonedSolutionsResponse,
-            MOCK_USER_SESSION,
-            templateDictionary
-          )
-          .then(() => done.fail(), done);
-      });
-
-      it("should handle error on get data", done => {
-        const _templates: any[] = [
-          {
-            type: "Group",
-            itemId: "NEWABC123",
-            dependencies: []
-          },
-          {
-            type: "Workforce Project",
-            itemId: "NEW123ABC",
-            dependencies: []
-          }
-        ];
-
-        const templateDictionary: any = {
-          unresolved: {
-            itemId: "resolved"
-          }
-        };
-
-        const workforceData: any = {
-          unresolvedVariable: "{{unresolved.itemId}}"
-        };
-
-        const clonedSolutionsResponse: common.ICreateItemFromTemplateResponse[] = [
-          {
-            id: "NEWABC123",
-            type: "Group",
-            postProcess: false
-          },
-          {
-            id: "NEW123ABC",
-            type: "Workforce Project",
-            postProcess: true
-          }
-        ];
-
-        fetchMock.post(
-          utils.PORTAL_SUBSET.restUrl + "/content/items/NEW123ABC/data",
-          mockItems.get400Failure()
-        );
-
-        deploySolution
-          .postProcessDependencies(
-            _templates,
-            clonedSolutionsResponse,
-            MOCK_USER_SESSION,
-            templateDictionary
-          )
-          .then(done.fail, done);
-      });
-    }
-  });
+  // describe("postProcessDependencies", () => {
```

Comment:
No longer needed?

Reply:
tbh - I was not clear around this, as that's a massive set of tests, which I've seemingly replaced (at least in terms of coverage) with way fewer... I wanted to keep this around in case I run into some issue where that code would be helpful... I'll remove it w/ #347 PR